thorlcr/activities/keyedjoin/thkeyedjoinslave.cpp

```diff
@@ -1826,6 +1817,7 @@ class CKeyedJoinSlave : public CSlaveActivity, implements IJoinProcessor, implem
                 activity.inactiveStats.mergeStatistic(StNumDiskSeeks, diskSeeks);
             };
             COnScopeExit scoped(onScopeExitFunc);
+            CStatsScopedThresholdDeltaUpdater scoped2(activity.statsUpdater);
```

Comment:
trivial: 'scoped2' makes it look like there's another use case in the same scope, better to remove the '2' here.

Reply:
There are 2 scopes COnScopeExit and CStatsScopedThresholdDeltaUpdater